implementation/contracts/deposit/DepositFunding.sol

```diff
@@ -40,12 +40,12 @@ library DepositFunding {
         _d.signingGroupPubkeyY = bytes32(0);
     }
 
-    /// @notice         The system can spin up a new deposit
-    /// @dev            This should be called by an approved contract, not a developer
-    /// @param _d       deposit storage pointer
-    /// @param _m       m for m-of-n
-    /// @param _m       n for m-of-n
-    /// @return         True if successful, otherwise revert
+    /// @notice         The system can spin up a new deposit.
+    /// @dev            This should be called by an approved contract, not a developer.
+    /// @param _d       deposit storage pointer.
+    /// @param _m       m for m-of-n.
+    /// @param _m       n for m-of-n.
```

Comment:
For m-of-n what? We can use more info on `can spin up a new deposit`. `@notice` should answer the question “What happens when I call this?” The answer can't be “The system can spin up a new deposit.“ The answer *could* be, “Spins up a new deposit with ... ”

This is also meant to be called internally. `@notice` is meant for [*end users*](https://solidity.readthedocs.io/en/develop/natspec-format.html#tags); the most we should tell an end user here is “don't call this, call `DepositFactory.createNewDeposit` instead!”.

For a developer, we probably want to tell them what happens if they do call it, so they know not to.